### ISSUE TITLE
Failing Examples

### ISSUE THREAD

**Issue by mainakjas** [mne-tools/mne-python#689]
I have failing examples when I compile the htmls. Help!

Here is the first traceback:

``` python
Traceback (most recent call last):
  File "/home/mainakjas/github/mne-python/doc/sphinxext/gen_rst.py", line 650, in generate_file_rst
    execfile(os.path.basename(src_file), my_globals)
  File "plot_channel_epochs_image.py", line 72, in <module>
    vmax=250, colorbar=True, order=order_func, show=True)
  File "/home/mainakjas/.local/lib/python2.7/site-packages/mne/viz.py", line 1998, in plot_image_epochs
    this_order = order(epochs.times, this_data)
  File "plot_channel_epochs_image.py", line 65, in order_func
    n_components=1, random_state=0).ravel())
  File "/usr/local/lib/python2.7/dist-packages/sklearn/manifold/spectral_embedding.py", line 259, in spectral_embedding
    tol=eigen_tol)
  File "/usr/local/lib/python2.7/dist-packages/scipy/sparse/linalg/eigen/arpack/arpack.py", line 1536, in eigsh
    symmetric=True, tol=tol)
  File "/usr/local/lib/python2.7/dist-packages/scipy/sparse/linalg/eigen/arpack/arpack.py", line 1037, in get_OPinv_matvec
    return LuInv(A).matvec
  File "/usr/local/lib/python2.7/dist-packages/scipy/sparse/linalg/eigen/arpack/arpack.py", line 932, in __init__
    self.M_lu = lu_factor(M)
  File "/usr/local/lib/python2.7/dist-packages/scipy/linalg/decomp_lu.py", line 58, in lu_factor
    a1 = asarray_chkfinite(a)
  File "/usr/local/lib/python2.7/dist-packages/numpy/lib/function_base.py", line 590, in asarray_chkfinite
    "array must not contain infs or NaNs")
ValueError: array must not contain infs or NaNs
```

Here is the second traceback:

``` python
Traceback (most recent call last):
  File "/home/mainakjas/github/mne-python/doc/sphinxext/gen_rst.py", line 650, in generate_file_rst
    execfile(os.path.basename(src_file), my_globals)
  File "plot_ssp_projs_sensitivity_map.py", line 38, in <module>
    ssp_ecg_map.plot(subject='sample', time_label='ECG SSP sensitivity', **args)
  File "/home/mainakjas/.local/lib/python2.7/site-packages/mne/source_estimate.py", line 1384, in plot
    subjects_dir=subjects_dir, figure=figure)
  File "/home/mainakjas/.local/lib/python2.7/site-packages/mne/viz.py", line 1623, in plot_source_estimates
    config_opts=config_opts, subjects_dir=subjects_dir)
  File "/usr/local/lib/python2.7/dist-packages/surfer/viz.py", line 197, in __init__
    self._geo.load_geometry()
  File "/usr/local/lib/python2.7/dist-packages/surfer/io.py", line 493, in load_geometry
    self.coords, self.faces = read_geometry(surf_path)
  File "/usr/local/lib/python2.7/dist-packages/surfer/io.py", line 71, in read_geometry
    with open(filepath, "rb") as fobj:
IOError: [Errno 2] No such file or directory: '/data/freesurfer/subjects/sample/surf/rh.inflated'
```

where do I get the missing file from?

and lastly something related to unicode characters: 

``` python
Exception occurred:
  File "/home/mainakjas/github/mne-python/doc/sphinxext/gen_rst.py", line 874, in embed_code_links
    line = line.replace(name, link)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 244: ordinal not in range(128)
```

**Comment by terminator9487** [comment]
Hi, I just got in touch with mne.decoding.csp Recently. the code i used is the same with demo (https://martinos.org/mne/stable/auto_examples/decoding/plot_decoding_csp_eeg.html） , and the data_t of Demo is a (45, 64, 161) matrix and the label_V of Demo is a (45,) matrix. Then, the demo works well.However, after applying this code to my data,which the data_t was changed to (1280, 32, 8064) matrix and label_V to (1280,) matrix. the same error that mentioned above was occurred. 
![screenshot from 2018-08-21 13-33-44](https://user-images.githubusercontent.com/13758812/44969931-41421e80-af8a-11e8-897e-981791898dd9.png)
![screenshot from 2018-08-21 13-34-34](https://user-images.githubusercontent.com/13758812/44969937-4f903a80-af8a-11e8-9ef6-2315b6815830.png)
the environment i worked is on the numpy 1.15.0 and sklearn  0.19.1  of Ubuntu Anaconda python 3.5. Is this problem related with the numpy or sklearn version？
 Does anyone know the reason of this error？ which is strange because the array of my data didn't include any infs or NaNs. Thanks a lot！
